pkg/config/testdata/containers_override.conf

```diff
@@ -1,6 +1,7 @@
 [containers]
 
 apparmor_profile = "overridden-default"
+interface_name = "device"
```

Comment:
This seem to be never used or checked anywhere?
You likely want to add this check to the `contents of passed-in file should override others` test.

Reply:
> This seem to be never used or checked anywhere?
> You likely want to add this check to the `contents of passed-in file should override others` test.

How about I revert this change as config_local_test.go already covers all scenarios?

Reply:
yes you can also remove it

Reply:
Done